tools/imports.EXAMPLE

```diff
@@ -26,6 +26,13 @@
 #   from bar import symbol_A
 #
 # and foo.symbol_A and bar.symbol_A do not point to the same object,
-# an error will be raised.
+# an error will be raised. Use the renaming capability to avoid
+# such naming conflicts; for instance,
+#
+#   from foo import symbol_A as symbol_A_foo
+#   from bar import symbol_A as symbol_A_bar
+#
+# The following import is 100% redundant; AXL includes an internal
+# import for this already. It is provided simply as an example.
 
 from axl.methods import *
```

Comment:
I'm actually thinking about reworking this. The reason I have it is so that the Excel convenience functions that transmit data to Python in certain formats can have access to them. But since they're hardcoded in VBA anyway, I see no reason why they can't use module syntax; e.g., `axl.methods.TupleVec` instead of just `TupleVec`. The user won't see any difference.

I also think we might as well import `math`, `numpy`, and `pandas` by default, for convenience.

Reply:
Would this cause namespace issues in existing spreadsheets? Lots of them make direct calls to "DFCols", i.e. `=DCols(x, y)` and other functions.

Reply:
No, it wouldn't---because the VBA function `DFCols` ends up just doing something like `P("DFCols"...)` internally. I can change that string. So it would require modifying the VBA code but that's fine.